src/lib/viewers/BaseViewer.js

```diff
@@ -214,6 +214,10 @@ class BaseViewer extends EventEmitter {
             });
         }
 
+        if (this.options.enableAnnotationsDiscoverability) {
+            this.rootEl.classList.add('bp-annotations-discoverable');
```

Comment:
Moved to BaseViewer because we want to add `bp-annotations-discoverable` to `bp` rather than `bp-doc`.

Reply:
Maybe `this.containerEl` might make more sense than the `rootEl` of preview

Reply:
Also may need to handle cleanup of the the class in the `destroy`